src/Gamepage/Gamepage.js

```diff
@@ -8,30 +8,73 @@ export class Gamepage extends Component {
     this.state = {
       hints: [],
       correctGuesses: [],
-      attempts: []
+      attempts: [],
+      currentGuess: ''
     }
   }
 
-  render() {
+  updateChange = (e) => {
+    e.preventDefault();
+    this.setState({currentGuess: e.target.value})
+  }
+
+
+  makeGuess = (e) => {
+    e.preventDefault();
+    // console.log("WordSelector -> makeWordToGuess -> this.state.guessWord", this.state.guessWord)
+    this.props.makeGuess(this.state.currentGuess)
+  }
+
+  mapAttempts = () => {
+    return this.props.attempts.map(attempt => {
+      return <p>{attempt}</p>
+    })
+  }
+
+  splitDisplay = () => {
+    return this.props.display.map(tile => {
+      return <p>{tile}</p>
```

Comment:
Make sure to add a `key` prop to these `p` tags or else React will be angry. They need to be unique also T.T

Reply:
gotcha, I added this as an issue